AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -462,11 +540,22 @@ - (BOOL)isPlaying
   return (_player.rate > 0 && !_player.error);
 }
 
+- (BOOL)isStateChangeValid:(ASVideoNodePlayerState)state
```

Comment:
In stead of checking the delegate respondsToSelector each time we play and pause, can we store this variable away in a property, something like 
_asyncDelegateFlags.asyncDelegateScrollViewDidScroll
Check out ASCollectionView.mm :)